feature/home/src/main/java/com/epmedu/animeal/home/presentation/viewmodel/HomeViewModel.kt

```diff
@@ -122,12 +123,30 @@ class HomeViewModel @Inject constructor(
         }
     }
 
-    private fun showWillFeedDialog() {
-        updateState { copy(willFeedState = WillFeedState(true)) }
+    private fun hideAllFeedingPointsButOne(selectedFeedingPoint: FeedingPointModel) {
+        updateState {
+            copy(feedingPoints = persistentListOf(selectedFeedingPoint))
+        }
     }
 
-    private fun dismissWillFeedDialog() {
-        updateState { copy(willFeedState = WillFeedState(false)) }
+    private fun saveUserAsCurrentFeeder(feedingPointId: Int) {
+        viewModelScope.launch {
+            saveUserAsFeederUseCase(feedingPointId).collect { isSuccessful ->
+                if (isSuccessful) {
```

Comment:
Could you pls add else branch?
Also, looks like with the current implementation it's really hard to know if something was wrong and an exception was thrown. 
Maybe, we should add logs to the SaveUserAsFeederUseCase.invoke method in case of error?

Reply:
Yeah no problem, @Fileek  what class are we using for logging?

Reply:
> Yeah no problem, @Fileek what class are we using for logging?

We use `Log` from `android.util` for now